spec/requests/api/v1/allocations_controller_index_spec.rb

```diff
@@ -13,7 +13,7 @@
   describe 'GET /allocations' do
     let(:schema) { load_yaml_schema('get_allocations_responses.yaml') }
 
-    let!(:allocations) { create_list :allocation, 21 }
+    let!(:allocations) { create_list :allocation, 2 }
```

Comment:
It feels silly to create 21 allocations for all of the specs here, it's only the pagination stuff that really needs quite so many so this should optimise the test performance a bit.